src/foraging_gui/Dialogs.py

```diff
@@ -3413,7 +3413,7 @@ def _connect_auto_training_manager(self):
                 self.MainWindow,
                 "Box {}, Error".format(self.MainWindow.box_letter),
                 "AWS connection failed!\n"
-                "Please check your AWS credentials at ~\.aws\credentials and restart the GUI!\n\n"
+                "Please check your AWS credentials at ~/.aws/credentials and restart the GUI!\n\n"
```

Comment:
This is wrong. Since we only run on windows it will be '\' not '/'

Reply:
`'\c'` is an invalid escape sequence. I changed it to `/` since i'm lazy. Would you prefer `\\c`?